test/TestDestroy.cpp

```diff
@@ -27,17 +27,27 @@ TEST(TestDestroy, TestDestroyTensorSingle)
         {
             kp::Manager mgr;
 
-            tensorA = mgr.tensor({ 0, 0, 0 });
+            const std::vector<float> initialValues = {0.0f, 0.0f, 0.0f};
+
+            tensorA = mgr.tensor(initialValues);
```

Comment:
This looks ok - just as a heads up you can have the same typed behaviour using `mgr.tensorT<float>(...)`, but you do use the EXPECT_EQ so it makes sense to separete these